src/components/Todo.jsx

```diff
@@ -16,19 +18,19 @@ export default function Todo({
 
   if (isEditing) {
     return (
-      <div>
+      <div className= "editing">
```

Comment:
Should we use nouns and not verbs for the classes? Editing could be very confusing, Maybe something like `edit-template-container`. But you think too, let's brain storm.

Reply:
good to know